.github/workflows/reusable-deployment.yml

```diff
@@ -31,12 +31,12 @@ jobs:
 
       - uses: actions/download-artifact@v4
         with:
-          name: helm-artifacts
+          name: helm-artifacts-[OS.ubuntu-latest]-[python-v.3.8]
```

Comment:
This values shouldn't be hardcoded

Reply:
@YuriiSmolii 
Good Day!
Please find below link for successfull workflow with branch pull request approve NOT requested:
https://github.com/YegorVolkov/devops_todolist_cicd_task_6_polish_pipeline/actions/runs/10228145227

Please find below screenshot showing the commit hashes prooving that no code changes were done compared to the state of the `current dev branch pull_request`
![1](https://github.com/user-attachments/assets/3b78eddf-6c23-4b64-9014-28c1c656bc79)

Please find below screenshot showing that staging env deployment is waiting for approval
![image](https://github.com/user-attachments/assets/3d18ece7-aa29-4532-980e-972a91f5f260)